.aws/.docker/agent/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM prefecthq/prefect:2-python3.10
+
+RUN pip install git+https://github.com/Pocket/prefect-aws.git@test-prs
```

Comment:
Due to the following issue, we need the agent to use our custom version:

- https://github.com/PrefectHQ/prefect-aws/pull/236

will switch to original once changes are merged and released